packages/account/src/Sections/Profile/PhoneVerification/resend-code-timer.tsx

```diff
@@ -0,0 +1,39 @@
+import React, { useRef } from 'react';
+import { Button, CaptionText } from '@deriv-com/quill-ui';
+import { Localize } from '@deriv/translations';
+
+const ResendCodeTimer = () => {
+    const [timer, setTimer] = React.useState(60);
+    const [startTimer, setStartTimer] = React.useState(false);
+    const resendCodeText = useRef('Resend code');
```

Comment:
We need to pass this text in as props or we can handle condition here cause the text for email verification is `Resend code` but the text for phone verification it's `Didn't get a code?`

Reply:
Yes, suisin